build/common/installer/scripts/tomlparser-prom-customconfig.rb

```diff
@@ -47,6 +47,10 @@
 
 @disableRSTelegraf = false
 
+def get_command_windows(env_variable_name, env_variable_value)
```

Comment:
function name is a little confusing? get_command_windows and it's doing set operations inside therefore not very readable. Maybe rename to something like construct_command_windows?

Reply:
We have the same name in other toml parser files as well. Wanted to keep it consistent